prediction_market_agent/agents/safe_guard_agent/safe_api_models/transactions.py

```diff
@@ -7,58 +7,61 @@
 class Token(BaseModel):
     address: ChecksumAddress
     decimals: int
-    logoUri: str
-    name: str
-    symbol: str
-    trusted: bool
-
-
-class Sender(BaseModel):
-    value: str
-    name: str | None
-    logoUri: str | None
+    logoUri: str | None = None
+    name: str | None = None
+    symbol: str | None = None
+    trusted: bool | None = None
+    imitation: bool | None = None
 
 
-class Recipient(BaseModel):
+class Address(BaseModel):
     value: str
-    name: str | None
-    logoUri: str | None
+    name: str | None = None
+    logoUri: str | None = None
```

Comment:
Why do we need the logoUri for the processing logic?

Reply:
It just was in the responses, so auto-generation included it here.